cob_moveit_config/cob4-7/.setup_assistant

```diff
@@ -1,6 +1,6 @@
 moveit_setup_assistant_config:
   URDF:
     package: cob_hardware_config
-    relative_path: cob4-7/urdf/cob4-7.urdf.xacro
+    relative_path: robots/cob4-7/urdf/cob4-7.urdf.xacro
   SRDF:
     relative_path: config/cob4-7.srdf
```

Comment:
why do you keep only for cob_moveit_config the structure `package_name/$(arg robot)/...`